File: 1721.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    double x1, y1, x2, y2;
    cin >> x1 >> y1 >> x2 >> y2;

    double result;
    double temp;
    temp = pow(x1 - x2, 2) + pow(y1 - y2, 2);
    result = sqrt(temp);

    cout << fixed;
    cout.precision(2);
    cout << result;

    return 0;
}

File: 1721.py
import math

x1, y1 = input().split()
x2, y2 = input().split()
x1 = eval(x1)
y1 = eval(y1)
x2 = eval(x2)
y2 = eval(y2)
#x1 = float(input())
#y1 = float(input())
#x2 = float(input())
#y2 = float(input())

result = format(math.sqrt(math.pow(x1 - x2, 2) + math.pow(y1 - y2, 2)), ".2f")

print(result)
